memmove_linux_arm64.go

```diff
@@ -0,0 +1,21 @@
+// Package pcap is a wrapper around the pcap library.
```

Comment:
Do you need this for arm64? Do you have reasons to believe that memmove is safer and performs better on arm64? I think we should get rid of this file for arm64 and not worry about memcpy or memmove being used.

Reply:
The driver for this change was compilation error on arm64 

github.com/lacework/pcap
/tmp/go-build576117789/github.com/lacework/pcap/_obj/pcap.cgo2.o: In function `__wrap_memcpy':
../../pcap/pcap.go:21: undefined reference to `memmove@GLIBC_2.2.5'
/usr/bin/ld: $WORK/github.com/lacework/pcap/_obj/cgo.o: No symbol version section for versioned symbol `memmove@GLIBC_2.2.5'